BlockSettleUILib/BSTerminalMainWindow.cpp

```diff
@@ -1368,6 +1368,7 @@ void BSTerminalMainWindow::openAuthManagerDialog()
 void BSTerminalMainWindow::openAuthDlgVerify(const QString &addrToVerify)
 {
    const auto showAuthDlg = [this, addrToVerify] {
+      authAddrDlg_->setModal(true);
```

Comment:
Do we really need modality here? I mean, I guess no action in this dialog requires the rest of the interface to be blocked.

Reply:
Also it is not required for a user to interact with any other dialog in the app if he went to manage auth addresses. Also it is really easy to lose this dialog in a background.